src/Inquirer/Validators/RangeValidator.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Microsoft.Win32.SafeHandles;
+using System;
+
+namespace InquirerCore.Validators
+{
+    public class RangeValidator : IValidator
+    {
+    private int iMin { set; get; }
+    private int iMax { set; get; }
+
+        public RangeValidator(int _iMin, int _iMax)
```

Comment:
For parameter naming we use lower camelCase, and don't use underscore `_`.

Reply:
ok.